gtsfm/averaging/rotation/rotation_averaging_base.py

```diff
@@ -35,18 +35,18 @@ def get_ui_metadata() -> UiMetadata:
 
     # ignored-abstractmethod
     @abc.abstractmethod
-    def run(
+    def run_rotation_averaging(
         self,
         num_images: int,
         i2Ri1_dict: Dict[Tuple[int, int], Optional[Rot3]],
-        i2Ti1_priors: Dict[Tuple[int, int], PosePrior],
+        relative_pose_priors: Dict[Tuple[int, int], PosePrior],
```

Comment:
Thanks for the PR, Ayush. Can we switch `relative_pose_priors` back to `i2Ti1_priors`? I think otherwise the ordering gets too confusing and it can lead to bugs.

Reply:
Dont we change it to i1Ti2 in Hilti?

Reply:
Not sure. My point as to why we need to be explicit : - )

In addition to using explicit naming, can we add a unit test for this, so we know whether it should be i2Ti1 or i1Ti2?

Reply:
Yes we switched to i1Ti2 too. I agree that having the explicit naming convention helps.

Reply:
Added one unit test too.